### PR TITLE
Fast fail in CI

### PR DESCRIPTION
**Pull request by ereslibre**
If we get a failure or a success in CI make sure we do as early as possible,
without the need to explicitly wait the timeout when there's already a
final status (we only timeout if we are actually spinning).

### CODE REVIEW

**Comment by ereslibre** [comment]
Rebased.

**Comment by ereslibre** [comment]
Do not merge yet, it needs to be polished, it gives false positives sometimes.

File: velum-bootstrap/spec/features/02-bootstrap-cluster.rb [hunk 1]
```diff
@@ -128,9 +128,17 @@
...
-        expect(page).to have_css(".fa-check-circle-o", count: node_number, wait: orchestration_timeout)
+        expect(page).to have_css(".fa-check-circle-o, .fa-times-circle", count: node_number, wait: orchestration_timeout)
       end
     end
     puts "<<< Orchestration completed"
+
+    puts ">>> Orchestration succeeded"
+    with_screenshot(name: :orchestration_succeeded) do
+      within(".nodes-container") do
+        expect(page).to have_css(".fa-check-circle-o", count: node_number)
```
Comment (vitoravelino) [inline velum-bootstrap/spec/features/02-bootstrap-cluster.rb]: The default is set to 3 minutes, I would use `wait: 5` here.

Comment (ereslibre) [reply velum-bootstrap/spec/features/02-bootstrap-cluster.rb]: Addressed 👍

**Pull request merged**